### ISSUE TITLE
Require Group Passphrase

### ISSUE THREAD

**Issue by qalqi** [kirillzyusko/react-native-wifi-p2p#9]
Hi, 
This is a great library. Once a group is created, we need passphrase from it.

Currently, only way to get it is manually and not programmatically from i.e Settings -> WIFI Direct -> Passphrase

Here is a snippet shared by android developer https://developer.android.com/training/connect-devices-wirelessly/wifi-direct#java
```java
manager.requestGroupInfo(channel, new GroupInfoListener() {
  @Override
  public void onGroupInfoAvailable(WifiP2pGroup group) {
      String groupPassword = group.getPassphrase();
  }
});
```

**Comment by kirillzyusko** [comment]
Hi, @qalqi
Yes, you are absolutely right
I will implement this method as soon as possible
Do I correct understand, that you need a method in JS, which will return `groupPassword` from code snippet?

**Comment by qalqi** [comment]
Raising PR. Please review it.

**Comment by kirillzyusko** [comment]
@qalqi May I ask you why do we need to wait ~3 s in order to call to another function?

**Comment by qalqi** [comment]
creategroup.then resulting in null password. If called after a second or so after creating a group, I am receiving group passphrase.

**Comment by kirillzyusko** [comment]
It's a little bit strange
But I think we can move forward even with this

**Comment by turingpavan** [comment]
Tx

**Comment by qalqi** [comment]
Tx

**Comment by kirillzyusko** [comment]
@qalqi I published new version (0.6.7) with your changes :)
